src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -15,6 +15,7 @@ import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
 import { getGreatPrimer } from '@bbc/gel-foundations/typography';
 
 import { ServiceContext } from '#contexts/ServiceContext';
+import getExternalLinkLang from '#lib/utilities/getExternalLinkLang';
```

Comment:
I'd place this file beside the component file since it's not a reusable function. Means if we delete the component then we delete the function that uses it instead of maintaining it and its tests when nothing uses it.

Reply:
Makes sense @jroebu14. It's done :)